config/core/rootfs-configs.yaml

```diff
@@ -51,6 +51,49 @@ rootfs_configs:
     script: "scripts/install-bootrr.sh"
     test_overlay: "overlays/baseline"
 
+  bullseye-igt:
+    rootfs_type: debos
+    debian_release: bullseye
+    arch_list:
+        - amd64
+        - arm64
+        - armhf
+        - mips
```

Comment:
We know there's a problem with the plain bullseye build for mips, and as far as I know there's no platform with this architecture running IGT tests.  So maybe we could just drop it for bullseye-igt?

Reply:
Okay, dropped `mips` arch from this PR